deploy/monitor/push_service_monitor.yaml

```diff
@@ -7,8 +7,8 @@ metadata:
   name: unifiedpush
 spec:
   endpoints:
-    - path: /api/metrics
-      port: server
+    - path: /rest/prometheus/metrics
+      port: web
   selector:
     matchLabels:
-      app: unifiedpush
\ No newline at end of file
+      app: example-unifiedpushserver
```

Comment:
while this works, it will generate a few targets that will never be used (for example, against oauth-proxy, postgres etc). We can change it to `service: <service-name>` so that there will be only 1 valid rule generated (just against the ups server)

Reply:
Excellent spot/tip .. all done.